suite/compatibility/json_rpc_test.go

```diff
@@ -0,0 +1,140 @@
+package compatibility
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+
+	"github.com/ethereum/go-ethereum/rpc"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/smartcontractkit/integrations-framework/config"
+	"github.com/smartcontractkit/integrations-framework/utils"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+type Result struct {
+	Name   string      `json:"name"`
+	Schema interface{} `json:"schema"`
+}
+
+type Method struct {
+	Name    string        `json:"name"`
+	Summary string        `json:"summary"`
+	Params  []interface{} `json:"params"`
+	Result  Result        `json:"result"`
+}
+
+type OpenRPCStruct struct {
+	Openrpc string      `json:"openrpc"`
+	Info    interface{} `json:"info"`
+	Methods []Method    `json:"methods"`
+}
+
+type RPCMethodCalls map[string][]interface{}
+
+var rpcMethodCalls RPCMethodCalls
+
+var _ = Describe("Json RPC compatibility @json_rpc", func() {
+	var (
+		openrpcSchema OpenRPCStruct
+		methods       []Method
+	)
+	rpcClientsByChain := make(map[int][]*rpc.Client)
+
+	jsonFile, err := os.Open(filepath.Join(utils.ProjectRoot, "openrpc.json"))
+	Expect(err).ShouldNot(HaveOccurred())
+	defer func(jsonFile *os.File) {
+		_ = jsonFile.Close()
+	}(jsonFile)
+
+	byteValue, _ := ioutil.ReadAll(jsonFile)
+	err = json.Unmarshal(byteValue, &openrpcSchema)
+	Expect(err).ShouldNot(HaveOccurred())
+
+	methods = openrpcSchema.Methods
+
+	rpcMethodCalls = RPCMethodCalls{
+		"eth_getBlockByNumber": []interface{}{"0x000000"},
+		"eth_chainId":          []interface{}{},
+		"eth_gasPrice":         []interface{}{},
+		"eth_getBalance":       []interface{}{"0x0000000000000000000000000000000000000000"},
+		"eth_getCode":          []interface{}{"0x0000000000000000000000000000000000000000"},
+		"eth_getLogs": []interface{}{map[string]interface{}{
+			"fromBlock": "0x000000",
+			"toBlock":   "0x000010",
+			"address":   "0x0000000000000000000000000000000000000000",
+		}},
+	}
+
+	BeforeEach(func() {
+		By("Getting RPC clients", func() {
+			nc, err := config.LoadNetworksConfig(filepath.Join(utils.ProjectRoot, "networks.yaml"))
+			Expect(err).ShouldNot(HaveOccurred())
+			for _, networkName := range nc.SelectedNetworks {
+				networkSettings, ok := nc.NetworkSettings[networkName]
+				Equal(ok)
```

Comment:
It doesn't seem clear what this is checking or if it's really asserting anything. Does `Equal` run an assert on `ok` this way? Doesn't it need to have something to check what it's equal against?

Reply:
It's like assert.True(...), but I'll try to make it more obvious, or will remove this assertion.

Reply:
Ah okay, that's fine then. I wasn't aware of that behavior.